pkg/machinery/client/resolver/roundrobin.go

```diff
@@ -62,9 +63,21 @@ func (r *roundRobinResolver) start() error {
 	var addrs []resolver.Address //nolint:prealloc
 
 	for _, a := range strings.Split(r.target.Endpoint, ",") { //nolint:staticcheck
+		addr := net.FormatAddress(a)
```

Comment:
we should stop using this one and use `stdnet.JoinHostPort` instead of this and `fmt.Sprintf(%s:%d)`